custom/icds_reports/tasks.py

```diff
@@ -1326,3 +1326,42 @@ def _child_health_monthly_aggregation(day):
 
     with get_cursor(ChildHealthMonthly) as cursor:
         cursor.execute(helper.drop_temporary_table())
+
+
+@task
+def email_location_changes(domain, old_location_blob_id, new_location_blob_id):
+    old_params = {
+        'file_format': 'csv',
+        'data_type': 'old_location_definitions',
```

Comment:
`data_type` should really be "filename for download", but would need to figure out a migration path for all those emails/files already sent